package.json

```diff
@@ -38,9 +40,12 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "prepublish": "npm test",
+    "prepublish": "npm run check-links && npm test",
     "test": "npm run lint && node bin/jest.js && npm run jasmine1",
-    "jasmine1": "node bin/jest.js --testRunner='<rootDir>/testRunners/jasmine/jasmine1.js'"
+    "jasmine1": "node bin/jest.js --testRunner='<rootDir>/testRunners/jasmine/jasmine1.js'",
+    "check-links": "[ \"$NODE_ENV\" != production ] && npm run link-packages",
```

Comment:
I think the target is linux and osx, but maybe it would be worth while to add this for win32 too. I am checking the node_env because I think it should only run when working in the jest repository.